### DISCUSSION TITLE
Expire non-visible post types

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-future#358] [Feature Requests]
> I see it is possible to expire WooCommerce products. But is it possible to expire WooCommerce product variations?
> https://docs.woocommerce.com/document/variable-product/
> 
> For example, we sell jewellery in two different materials (Gold and Silver). We would like a period where we sell an additional material (Rose Gold). Is it possible to expire the product variations, Rose Gold, and keep the remaining two materials?

https://wordpress.org/support/topic/is-it-possible-to-expire-woocommerce-product-variations/

**Comment by stevejburge** [comment]
One more request for this.

**Comment by stevejburge** [comment]
A third request in that same WordPress.org opportunity
https://wordpress.org/support/topic/is-it-possible-to-expire-woocommerce-product-variations/

**Comment by masteradhoc** [comment]
+1

**Comment by rizaardiyanto1412** [comment]
Another request on HC ticket
